### DISCUSSION TITLE
Disabling motion detection

### DISCUSSION THREAD

**Discussion by JasonVarney** [roflcoopter/viseron#230] [Q&A]
Great job with this project, very much appreciated.

Is there an elegant way to completely disable all motion detection, at the moment I'm using a rather ugly hack of just setting the interval to 21600?

motion_detection:
  interval: 21600
  trigger_recorder: False
  trigger_detector: False

One (of 5) of the camera examples...

cameras:
   - name: FrontDoor
     width: 2304
     height: 1296
     fps: 15
     audio_codec: "aac"
     rtsp_transport: tcp
     mqtt_name: FrontDoor
     host: 192.168.1.201
     port: 8554
     username: xxxxxx
     password: xxxxxx
     path: /proxied1
     static_mjpeg_streams:
       frontdoor-debug:
         width: 2304
         height: 1296
         draw_objects: True
         draw_zones: True
     object_detection:
       enable: true
       labels:
         - label: person
           require_motion: False
           confidence: 0.9
           height_min: 0.232
           height_max: 0.9999
           width_min: 0.0695
           width_max: 0.434
           trigger_recorder: true

Yet I still see motion detection is initialising in the logs.

**Comment by roflcoopter** [comment]
Thanks for checking out Viseron!

Setting `trigger_detector: false` and `trigger_recorder: false` under `motion_detection` effectively disables motion detection so it will never run.

You will still see it setting up in the logs, but the no detection will ever run.
I recently added an `enable` flag to the object detection, i should probably do the same with motion detection to make this a bit more obvious

**Reply by JasonVarney** [reply]
Thank you for the clarification, amazing work.
